src/utils/wallet.js

```diff
@@ -870,6 +870,8 @@ class Wallet {
             if (lastAccount) {
                 store.dispatch(redirectTo(`/profile/${lastAccount.accountId}`, { globalAlertPreventClear: true }))
                 throw lastAccount.error
+            } else {
+                throw accountIdsError.reverse()[accountIdsError.length - 1].error
```

Comment:
@marcinbodnar why call `.reverse()` and then take last element? Seems like this can be just `accountIdsError[0].error`?

Reply:
@vgrichina I made it to be consistent with throwing `LackBalanceForState` error (first account) - but it's not necessary. Removed it.

Reply:
@marcinbodnar BTW I just realized that before fix it would also use wrong order, as `.reverse()` is happening in-place

Reply:
@vgrichina yeah, you're right